app/Blocks/AllowedBlocks.php

```diff
@@ -13,80 +13,80 @@ public function limitAllowedBlocks($allowed_block_types, $block_editor_context)
-				'core/audio',
-				'core/avatar',
-				'core/block',
+				// 'core/archives',
+				// 'core/audio',
+				// 'core/avatar',
+				// 'core/block',
 				'core/button',
 				'core/buttons',
-				'core/calendar',
-				'core/categories',
-				'core/code',
+				// 'core/calendar',
+				// 'core/categories',
+				// 'core/code',
 				'core/column',
 				'core/columns',
-				'core/cover',
+				// 'core/cover',
 				'core/embed',
 				'core/file',
 				'core/footnotes',
 				'core/freeform',
 				'core/gallery',
 				'core/group',
 				'core/heading',
-				'core/home-link',
-				'core/html',
+				// 'core/home-link',
+				// 'core/html',
 				'core/image',
-				'core/latest-comments',
-				'core/latest-posts',
+				// 'core/latest-comments',
+				// 'core/latest-posts',
 				'core/list',
 				'core/list-item',
 				'core/media-text',
-				'core/missing',
+				// 'core/missing',
 				'core/more',
-				'core/navigation',
-				'core/navigation-link',
-				'core/navigation-submenu',
-				'core/nextpage',
-				'core/page-list',
-				'core/page-list-item',
+				// 'core/navigation',
+				// 'core/navigation-link',
+				// 'core/navigation-submenu',
+				// 'core/nextpage',
+				// 'core/page-list',
+				// 'core/page-list-item',
 				'core/paragraph',
-				'core/pattern',
-				'core/post-author',
-				'core/post-author-biography',
-				'core/post-author-name',
-				'core/post-comments-count',
-				'core/post-comments-form',
-				'core/post-comments-link',
-				'core/post-content',
-				'core/post-date',
-				'core/post-excerpt',
+				// 'core/pattern',
+				// 'core/post-author',
+				// 'core/post-author-biography',
+				// 'core/post-author-name',
+				// 'core/post-comments-count',
+				// 'core/post-comments-form',
+				// 'core/post-comments-link',
+				// 'core/post-content',
+				// 'core/post-date',
+				// 'core/post-excerpt',
 				'core/post-featured-image',
-				'core/post-navigation-link',
-				'core/post-terms',
-				'core/post-time-to-read',
-				'core/post-title',
-				'core/preformatted',
-				'core/query',
-				'core/query-no-results',
-				'core/query-pagination',
-				'core/query-pagination-next',
-				'core/query-pagination-numbers',
-				'core/query-pagination-previous',
-				'core/query-title',
+				// 'core/post-navigation-link',
+				// 'core/post-terms',
+				// 'core/post-time-to-read',
+				// 'core/post-title',
+				// 'core/preformatted',
+				// 'core/query',
+				// 'core/query-no-results',
+				// 'core/query-pagination',
+				// 'core/query-pagination-next',
+				// 'core/query-pagination-numbers',
+				// 'core/query-pagination-previous',
+				// 'core/query-title',
 				'core/quote',
-				'core/read-more',
-				'core/rss',
-				'core/search',
+				// 'core/read-more',
+				// 'core/rss',
+				// 'core/search',
 				'core/separator',
-				'core/shortcode',
-				'core/site-logo',
-				'core/site-tagline',
-				'core/site-title',
-				'core/social-link',
-				'core/social-links',
+				// 'core/shortcode',
```

Comment:
This needs to be enabled, to allow continued use of the _Footnotes_ shortcode, if they prefer the existing workflow to the native block.